### PR TITLE
[Remove unused reference] LPS-166252 

### PR DESCRIPTION
**Pull request by JanisZhang**
This PR is to remove unused fields in` commerce-warehouse-web`. Thanks!
https://issues.liferay.com/browse/LPS-166252

### CODE REVIEW

**Comment by catarineaguiar** [comment]
ci:test:relevant

**Comment by jaydawu** [comment]
No regressions found.

cc/ @zhangyan0701

**Comment by zhangyan0701** [comment]
ci:test:commerce

**Comment by catarineaguiar** [comment]
@JanisZhang no regressions found.

File: modules/apps/commerce/commerce-warehouse-web/src/main/java/com/liferay/commerce/warehouse/web/internal/portlet/action/EditCommerceInventoryWarehouseItemMVCActionCommand.java [hunk 1]
```diff
@@ -111,7 +110,4 @@ protected void doProcessAction(
 	private CommerceInventoryWarehouseItemService
 		_commerceInventoryWarehouseItemService;
 
-	@Reference
-	private Portal _portal;
```
Comment (dantewang) [inline modules/apps/commerce/commerce-warehouse-web/src/main/java/com/liferay/commerce/warehouse/web/internal/portlet/action/EditCommerceInventoryWarehouseItemMVCActionCommand.java:115]: @JanisZhang Please record this one, `SessionErrors` has a lot of methods calling `PortalUtil`.

Comment (JanisZhang) [reply modules/apps/commerce/commerce-warehouse-web/src/main/java/com/liferay/commerce/warehouse/web/internal/portlet/action/EditCommerceInventoryWarehouseItemMVCActionCommand.java:115]: Got it, thanks

**Pull request closed without merging**